build.gradle

```diff
@@ -6,18 +6,19 @@ buildscript {
 	dependencies {
 		classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
 		classpath "org.jetbrains.kotlin:kotlin-allopen:$kotlinVersion"
-		classpath 'org.asciidoctor:asciidoctorj-pdf:1.5.0-beta.6'
 	}
 }
 
 plugins {
 	id 'org.sonarqube' version '2.8'
-	id 'org.asciidoctor.convert' version '1.6.1'
 	id 'io.spring.nohttp' version '0.0.4.RELEASE' apply false
 	id 'org.ajoberstar.grgit' version '4.0.1'
 	id "io.spring.dependency-management" version '1.0.9.RELEASE'
 	id 'com.jfrog.artifactory' version '4.15.1' apply false
 	id 'org.jetbrains.dokka' version '0.10.1'
+	id 'org.asciidoctor.jvm.pdf' version '3.1.0'
+	id 'org.asciidoctor.jvm.gems' version '3.1.0'
+	id 'org.asciidoctor.jvm.convert' version '3.1.0'
```

Comment:
Can we extract this as a `asciidoctorVersion`?
Alongside with that `ext.kotlinVersion = '1.3.71'` above.

Thanks

Reply:
I don't think the asciidoctor folks are trying to keep them in sync. I think it's a coincidence that they are the same just now. We can do it, but we may end up with three variables down the road. I'll go ahead and do it, though, since it seems to be Spring's usual way of doing things.